content/compose/environment-variables/set-environment-variables.md

```diff
@@ -44,12 +48,31 @@ services:
     image: 'webapp:v1.5'
 ```
 
-The `.env` file should be placed at the root of the project directory next to your `compose.yaml` file. You can use an alternative path with one of the following methods:
-- The [`--file` option in the CLI](../reference/index.md#use--f-to-specify-name-and-path-of-one-or-more-compose-files) 
-- The [`--env-file` option in the CLI](#substitute-with---env-file)
-- Using the [`env_file` attribute in the Compose file](../compose-file/05-services.md#env_file)
-
-For more information on formatting an environment file, see [Use an environment file](env-file.md).
+#### Additional information 
+
+- If you define an environment variable in your `.env` file, you can reference it directly in your `compose.yml` with the [`environment` attribute](../compose-file/05-services.md#environment). For example, if your `.env` file contains the environment variable `DEBUG=1` and your `compose.yml` file looks like this:
+  ```yaml
+    services:
+      webapp:
+        image: my-webapp-image
+        environment:
+          - DEBUG=${DEBUG}
+  ```
+  Docker Compose replaces `${DEBUG}` with the value from the `.env` file
+- You can use multiple `.env` files in your `compose.yml` with the [`env_file` attribute](../compose-file/05-services.md#env_file), and Docker Compose reads them in the order specified. If the same variable is defined in multiple files, the last definition takes precedence:
+  ```yaml
+  services:
+    webapp:
+      image: my-webapp-image
+      env_file:
+        - .env
+        - .env.override
+  ```
+- You can place your `.env` file in a location other than the root of your project's directory, and then use one of the following methods so Compose can navigate to it:
```

Comment:
I think there might be some nuance missing here. The .env file can be referenced from an arbitrary location with the env_file attribute or --env-file flag (as described on line 73) but with the --file flag, I assume that the .env file needs to be placed adjacent to the compose file referenced by --file. You couldn't point to a .env file with -f